rsc/extra/ocpChecker/dune-project

```diff
@@ -0,0 +1 @@
+(lang dune 1.5)
```

Comment:
We don't need this file, as this should be part of the alt-ergo project.

Reply:
Fixed